test/testdata/deployednettemplates/recipes/scenario3s/genesis.json

```diff
@@ -2584,6 +2584,7 @@
   ],
   "FeeSink": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ",
   "RewardsPool": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ",
+  "RewardsPoolBalance": 125000000000000,
```

Comment:
Just curious, how come we're adding RewardsPoolBalance to scenario3s?

Reply:
make does it, idk why. probably the source changed before and make was never run